DAO.sol

```diff
@@ -516,7 +521,12 @@ contract DAO is DAOInterface, Token, TokenCreation {
         uint waitPeriod = p.newCurator
             ? splitExecutionPeriod
             : executeProposalPeriod;
-        // If we are over deadline and waiting period, assert proposal is closed
+                
+        // Add grace period to wait time if proposal appears passable
+        if (p.yea > p.nay)
+           waitPeriod += splittingGracePeriod;
```

Comment:
Is it possible for an attacker to call `executeProposal` repeatedly. Then each time `waitPeriod` will be increased?